onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -125,8 +126,71 @@ static Status ReleaseNodeMLValues(ExecutionFrame& frame,
 Status SequentialExecutor::Execute(const SessionState& session_state, const std::vector<int>& feed_mlvalue_idxs,
                                    const std::vector<OrtValue>& feeds, const std::vector<int>& fetch_mlvalue_idxs,
                                    std::vector<OrtValue>& fetches,
-                                   const std::unordered_map<size_t, CustomAllocator>& fetch_allocators,
-                                   const logging::Logger& logger) {
+                                   const std::unordered_map<size_t, IExecutor::CustomAllocator>& fetch_allocators,
+                                   const logging::Logger& logger,
+                                   int64_t& run_id) {
+  const SequentialExecutionPlan& seq_exec_plan = *session_state.GetExecutionPlan();
+  const auto& exec_plan_vec = seq_exec_plan.execution_plan;
+  const auto exec_plan_size = exec_plan_vec.size();
+  if (run_id == DEFAULT_RUN_ID) {
```

Comment:
It would be cleaner, thanks for the great suggestion. 

---
In reply to: [598972085](https://github.com/microsoft/onnxruntime/pull/7077#discussion_r598972085) [](ancestors = 598972085)